htdocs/variants/create.php

```diff
@@ -70,7 +70,7 @@
 
 print '<form method="POST" action="'.$_SERVER["PHP_SELF"].'">';
 print '<input type="hidden" name="token" value="'.newToken().'">';
-print '<input type="hidden" name="action" value="add">';
+print '<input type="hidden" name="action" value="create">';
```

Comment:
In other screen, we use the action=create to say "show me the form to create" and "add" to say "form is sent, make insert into database".
So i change by replacing at the action level create into add